### ISSUE TITLE
Problem with observation fields (SSobservationType, SSobservationActivityType, and FOobservationCode) for self-sampling

### ISSUE THREAD

**Issue by davidcurrie2001** [ices-tools-dev/RDBES#114]
I was helping the Faroes fill in some data from their self-sampling pelagic program and noticed that the mandatory fields SobservationType and SSobservationActivityType fields in the Species Selection table didn't really make sense when there isn't an observer present.  I think there is a similar issue with the field FOobservationCode.  
Should we use "NotRecorded" (from code lists RS_ObservationActivityType, RS_ObservationCode) in this case or is there a need for an "Unknown" code to be added to the code lists?  
Also, the code list RS_ObservationType does not currently have a "NotRecorded" (or equivalent) value - does this need to be added?

**Comment by edvinf** [comment]
We fitted a self-sampling program during WKRDB-POP3 where the ObservationActivityType made sense and is potentially clarifying. I generally found this tricky to fill in when catchReg is "Landed" for at-sea programs (which is often the case with self-sampling programs). In those cases I have opted for an observationActivityType code that indicates that only 'sorting' was observed, and put 'vol' for the observationType. This is suggested for on-shore sampling in the field documentation of SSobsActTyp.

**Comment by davidcurrie2001** [comment]
Yes, I think we could recommend something along those lines and fit the data in.  I think in the case of self-sampling programmes where there is no observation of the process (e.g. you request a sample and it is selected in whichever way the fisher chooses) then perhaps it doesn't accurately represent how the data was collected.  The sampler will be described as "self-sampled" so perhaps that will be clear enough when taken in combination with the other values.

**Comment by lizclarke** [comment]
The estimation subgroup discussed this on 07/09/2021. We conclude that the distinction between "NotRecorded" and "Unknown" is not relevant to estimation and so we pass this decision back to the Core Group. We note that "Unknown" is used more commonly within ICES vocab code lists and so could possibly be used instead of "NotRecorded". We agree with the proposal to add "NotRecorded" to what was called observationType and is now called ObservationMethod (subject to the discussions on the need for NotRecorded and Unknown). There is one code list where both codes exist and this also needs to be checked against these conclusions - this is BycatchMitigationDevice. We agree with the guidance proposed for landings sampled at-sea (to use sorting and vol for ObservationActivityType and ObservationMethod respectively) and propose this is added to the documentation.

**Comment by HenrikK-N** [comment]
For the code type ObservationActivityType and ObservationCode I have requested to rename the code ‘NotRecorded’ to ‘NotKnown', to be consistent with most other generic code types which use 'NotKnown'.
